content/configuration/access-private-git-submodules.md

```diff
@@ -8,13 +8,7 @@ If your project requires accessing any private Git submodules or dependencies, y
 
 1. [Create an SSH key pair](../knowledge-base/generating-an-ssh-key) for use with Codemagic. Note that the SSH key **cannot** be password-protected.
 2. Add the **public key** to your repository settings. See how to do that on [GitHub](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account), [Bitbucket](https://confluence.atlassian.com/bitbucket/use-access-keys-294486051.html), [GitLab](https://docs.gitlab.com/ee/ssh/README.html#adding-an-ssh-key-to-your-gitlab-account).
-3. Encrypt the contents of the **private key** file and add it as an [environment variable](../building/environment-variables/) in your Codemagic configuration file. Make sure to check **Secure**. For example:
-
-    ```
-    SSH_KEY = -----BEGIN OPENSSH PRIVATE KEY-----
-    ...
-    -----END OPENSSH PRIVATE KEY-----
-    ```
+3. Copy the contents of the **private key** file add it as an [environment variable](../building/environment-variables/) in your Codemagic configuration file. Make sure to mark the Secure checkbox which will encrypt the contents of the private key file.
```

Comment:
This is suggesting to add the environment variable directly to the codemagic.yaml. The link is also to the read only environment variables page instead of the page about environment variable groups. I would suggest trying to explain the steps i.e.
- Add an environment variable that ends with _SSH_KEY e.g. MYKEY_SSH_KEY, and add to a group
- Import the env var group as explain on the env variable groups page and link to it.